### PR TITLE
refactor: migrate to t3-env

### PR DESCRIPTION
**Pull request by juliusmarminge**
🤫

# Tests

## Edge

CleanShot 2023-04-24 at 00 20 03

## Lambda

CleanShot 2023-04-24 at 00 20 24

## Client

CleanShot 2023-04-24 at 00 21 03

CleanShot 2023-04-24 at 00 21 22

## Starting dev / build

CleanShot 2023-04-24 at 00 22 21

### CODE REVIEW

**Comment by c-ehrlich** [comment]
works in docker... ie it still respects `SKIP_ENV_VALIDATION`

File: cli/template/base/tsconfig.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "**/*.ts",
     "**/*.tsx",
     "**/*.cjs",
-    "**/*.mjs"
+    "**/*.mjs",
+    "../extras"
```
Comment (c-ehrlich) [inline cli/template/base/tsconfig.json:31]: what is this for?

File: cli/package.json [hunk 2]
```diff
@@ -52,6 +52,7 @@
   },
   "devDependencies": {
     "@prisma/client": "^4.11.0",
+    "@t3-oss/env-nextjs": "^0.1.0",
```
Comment (c-ehrlich) [inline cli/package.json:55]: why is this a lower version than in the template package.json?

Comment (juliusmarminge) [reply cli/package.json:55]: must have missed it...

File: cli/template/base/tsconfig.json (hunk 1 above)
Comment (juliusmarminge) [reply cli/template/base/tsconfig.json:31]: eehem... prob something that snuck in from an autoimport somewhere... will remove!

**Review by c-ehrlich — APPROVED** [review]
🚀

**Pull request merged**